src/Sarif.Driver/Sdk/AnalyzeCommandBase.cs

```diff
@@ -473,7 +473,7 @@ private HashSet<Skimmer<TContext>> CreateSkimmers(TContext context)
 
             try
             {
-                skimmers = DriverUtilities.GetExports<Skimmer<TContext>>(DefaultPlugInAssemblies);
+                skimmers = CompositionUtilities.GetExports<Skimmer<TContext>>(DefaultPlugInAssemblies);
```

Comment:
>CompositionUtilities [](start = 27, length = 20)

Renamed this class because it was in a file named CompositionUtilities.cs. I put my new helper method in a class named `DriverUtilities` in a same-named file.